src/components/LinkManager/LinkList/LinkOverview/LinkOverView.tsx

```diff
@@ -0,0 +1,49 @@
+import Link from "next/link";
+import styles from "./LinkOverview.module.scss";
+import { Link as LinkType } from "@/src/constants/types";
+import getElapsedTime from "@/src/utils/getElapsedTime";
+import getFormattedDate from "@/src/utils/getFormattedDate";
+import { LinkImageEmptyCase } from "@/src/asset/";
+import Image from "next/image";
+
+interface LinkOverview {
+  link: LinkType;
+}
+
+export default function LinkOverview({ link }: LinkOverview) {
+  const { imageSource: imageUrl, createdAt, description, title, url } = link;
+  const createDate = getElapsedTime(createdAt);
+  const formatDate = getFormattedDate(createdAt);
+
+  // 서버 Image-url이 문제가 있어서 그냥 null처리 했읍니다
+  function fixedUrl(url: string) {
+    if (!url) return null;
+
+    if (url.startsWith(`https:`)) {
+      return url;
+    } else {
+      return null;
+    }
```

Comment:
예외 조건을 먼저 체크해서 끝내버리고, 함수의 의도와 가장 가까운 행위를 해주는식으로 조건식을 작성하는게 좋습니다~
```suggestion
    if (!url.startsWith(`https:`)) {
      return null;
    }
    
    return url
```

Reply:
코드를 조금 더 수정해보면 if 조건식을 통합시켜봐도 좋을 것 같아요.
이렇게 표현할 수 있을 것 같네요~
```ts
if(url === '' || !url.startsWith('https:')) {
  return null;
}

return url;
```